README.md

```diff
@@ -1 +1,250 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This project will take you through how to create a branch and pull request.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to start this project, you need to be familiar with basic **git commands** and how to **create** and **clone** a repository.
+
+### Setup
+
+Clone this repository to your desired folder.
+
+Example commands:
+
+  **cd my-folder**
+  **git clone git@github.com:myaccount/my-project.git**
+
+Create your files in your add your code:
+
+Example command: **touch index.html**
+
+### Create a branch
+
+To create a branch and switch to it, execute the following command:
+
+Example command: **git checkout -b name-of-branch**
+
+Check whether you're currently in the new branch:
+
+Example command: **git branch**
+
+
+### Make changes
+
+Now you can make the changes you want in the new branch
+
+### Add, Commit and Push your changes. 
+
+Example command to push changes: **git push -u origin name-of-branch**
+  
+Visit your repo on github. Add a descriptive title and comments and click on **Create pull request** button.
+
+![pull-request](https://user-images.githubusercontent.com/97234029/226851031-3697492a-367e-4ee2-ae3b-e1cb4ad2cc2e.jpg)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Prosper Kessie</a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```

Comment:
- Please make sure to add at least 1 future feature for this project ✨as it shows your future clients you have a growth mindset by setting next steps to take 💯

Reply:
Resolved